toolchain.mk

```diff
@@ -23,9 +23,11 @@ SRC_AARCH64_GCC 		?= https://developer.arm.com/-/media/Files/downloads/gnu-a/10.
 # $(3) is $.._GCC_VERSION	: the name of the file to download
 define dltc
 	@if [ ! -d "$(1)" ]; then \
-		mkdir -p $(1); \
 		echo "Downloading $(3) ..."; \
-		curl -s -L $(2) -o $(TOOLCHAIN_ROOT)/$(3).tar.xz; \
+		{ curl --retry-all-errors --retry 5 -s -L $(2) -o $(TOOLCHAIN_ROOT)/$(3).tar.xz || \
```

Comment:
Is the first layer of braces `{ }` needed? I do not think so (`{ curl ...` => `curl ...`).

Reply:
Yes. That's only for readability to simplify reading and parsing. Remove it?

Reply:
Yes.